src/main/java/com/renatusnetwork/momentum/data/cmdsigns/CommandSign.java

```diff
@@ -0,0 +1,49 @@
+package com.renatusnetwork.momentum.data.cmdsigns;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class CommandSign {
+	private final String name;
+	private String command;
+	private final CmdSignLocation location;
+
+	private Set<String> usages; // set of uuids of players that have used this command sign
```

Comment:
I think it would be better to load this on the PlayerStats side. We want the data related to who to stay in the same place

Reply:
got it